jenkins/scripts/regression-report.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#Usage: regression-report.sh <last number of days>
+# if anything fails, we'll abort
+set -e
+yum install -y python-dateutil python-blessings python-requests python-lxml
+
+no_of_days=$1
+git clone https://github.com/gluster/glusterfs.git
+cd glusterfs/
+python extras/failed-tests.py --html_report get-summary $no_of_days centos netbsd > regression_report.html
```

Comment:
Pretty sure `--html_report` is not a valid flag for the command. It's been failing locally for me.